tools/getkeys/package.json

```diff
@@ -14,11 +14,21 @@
   "main": "index.js",
   "scripts": {
     "start": "node ./index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Policy check will fail unless these are sorted alphanumerically. You should be able to run the associated fix script to clean them up automatically.

Reply:
Fixed! Thanks for catching it up.